src/Models/Traits/Searchable.php

```diff
@@ -51,14 +33,4 @@ public function searchableAs(): string
             config('rapidez.store'),
         ]));
     }
-
-    protected function makeAllSearchableUsing(Builder $query)
-    {
-        return $query;
-    }
-
-    public function makeSearchableUsing(Collection $models): Collection
-    {
-        return $models;
-    }
```

Comment:
These functions were not used other than in the actual model. They could be removed.